### ISSUE TITLE
EX: examples.ex_pandas breaks earlier than the copy/pasted traceback

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#5690]
In trying to debug the traceback copy/pasted into examples, ex_pandas, I find running it fails before I get to that point.  For me it fails on line 93:

```
>>> model = tsa.VAR(log_difference, freq='D')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "statsmodels/tsa/vector_ar/var_model.py", line 519, in __init__
    super(VAR, self).__init__(endog, exog, dates, freq, missing=missing)
  File "statsmodels/tsa/base/tsa_model.py", line 52, in __init__
    self._init_dates(dates, freq)
  File "statsmodels/tsa/base/tsa_model.py", line 187, in _init_dates
    raise ValueError('The given frequency argument could'
ValueError: The given frequency argument could not be matched to the given index.
```

We should audit the examples directory and move non-working ones to sandbox.examples or something until they are fixed.

**Comment by jbrockmendel** [comment]
ex_generic_mle also breaks earlier than in the copy/pasted traceback, on line 29

```
>>> res = mod.fit(method="nm", fargs=(data.endog,data.exog), maxiter=500)
Optimization terminated successfully.
         Current function value: 0.400588
         Iterations: 292
         Function evaluations: 494
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "statsmodels/base/model.py", line 892, in fit
    disp=disp, callback=callback, **kwargs)
  File "statsmodels/base/model.py", line 474, in fit
    H = -1 * self.hessian(xopt)
  File "statsmodels/base/model.py", line 848, in hessian
    return approx_hess(params, self.loglike)
  File "statsmodels/tools/numdiff.py", line 351, in approx_hess3
    - (f(*((x - ee[i, :] + ee[j, :],) + args), **kwargs)
TypeError: probitloglike() takes exactly 3 arguments (1 given)
```

**Comment by jbrockmendel** [comment]
The docstring'd-out output in ex_generic_mle_t doesn't match what I'm getting when running the code.  Some of it, e.g. `tmp.shape` looks really weird.  The LinAlgError on line 256 isn't happening anymore.

**Comment by jbrockmendel** [comment]
The docstring'd-out error in sandbox.distributions.try_pot I think is caused on line 32 by setting `xthreshold = xsorted[np.floor(nobs * frac).astype(int)]` instead of the slice `xthreshold = xsorted[:np.floor(nobs * frac).astype(int)]`

**Comment by jbrockmendel** [comment]
Closed by #5717.

**Comment by bashtage** [comment]
I think the closes #xxxx  has to be in the first post. Or you can put it in the commit message.
